brickpi3/brickpi3.h

```diff
@@ -174,8 +140,12 @@ enum brickpi3_i2c_flags {
 	BRICKPI3_I2C_SAME	= 0x04,
 };
 
-#define BRICKPI3_I2C_MAX_WRITE_SIZE	16
-#define BRICKPI3_I2C_MAX_READ_SIZE	14
+enum brickpi3_motor_status {
+    BRICKPI3_MOTOR_STATUS_LOW_VOLTAGE_FLOAT = 0x01,
+};
+
+#define BRICKPI3_I2C_MAX_SET_SIZE	16
```

Comment:
I would rather not rename this constant. "get" and "set" are more high-level terms and and "read" and "write" are more low-level, generally speaking. I2C is more of a low level thing, so I would like to keep it as `BRICKPI3_I2C_MAX_WRITE_SIZE`